sdk/cosmos/pom.xml

```diff
@@ -1,72 +1,96 @@
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-data-sdk-parent</artifactId>
+    <version>1.2.0</version>
+    <relativePath>../../pom.data.xml</relativePath>
+  </parent>
+
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-cosmos-parent</artifactId>
   <version>3.0.0</version>
   <packaging>pom</packaging>
-  <name>Azure Cosmos DB SQL API</name>
-  <description>Java Async SDK (with Reactive Extension RX support) for Azure Cosmos DB SQL API</description>
-  <url>https://docs.microsoft.com/en-us/azure/cosmos-db</url>
-  <modules>
-    <module>sdk</module>
-    <module>benchmark</module>
-    <module>examples</module>
-  </modules>
+
+  <name>Microsoft Azure Cosmos DB SQL API</name>
+  <description>This package contains Mircosoft Azure SDK for Azure Cosmos DB SQL API (with Reactive Extension RX support)</description>
+  <url>https://github.com/Azure/azure-sdk-for-java</url>
+
+  <distributionManagement>
+    <site>
+      <id>azure-java-build-docs</id>
+      <url>${site.url}/site/${project.artifactId}</url>
+    </site>
+  </distributionManagement>
+
+  <scm>
+    <url>https://github.com/Azure/azure-sdk-for-java</url>
+  </scm>
+
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-    <assertj.version>3.11.1</assertj.version>
-    <commons-io.version>2.5</commons-io.version>
-    <commons-lang3.version>3.8.1</commons-lang3.version>
-    <commons-text.version>1.6</commons-text.version>
-    <commons-validator-version>1.6</commons-validator-version>
-    <commons-validator.version>1.6</commons-validator.version>
     <cosmosdb-sdk-direct-impl.version>3.0.0</cosmosdb-sdk-direct-impl.version>
     <direct-connectivity-version>3.0.0</direct-connectivity-version>
-    <guava.version>27.0.1-jre</guava.version>
-    <hamcrest.version>1.3</hamcrest.version>
-    <jackson-databind.version>2.9.8</jackson-databind.version>
-    <java-uuid-generator.version>3.1.4</java-uuid-generator.version>
-    <jcommander.version>1.58</jcommander.version>
-    <log4j.version>1.2.17</log4j.version>
-    <metrics.version>4.0.5</metrics.version>
     <mockito.version>1.10.19</mockito.version>
     <netty.version>4.1.36.Final</netty.version>
-    <netty-tcnative.version>2.0.25.Final</netty-tcnative.version>
-    <reactor-addons.version>3.2.2.RELEASE</reactor-addons.version>
-    <reactor-bom.version>Californium-SR7</reactor-bom.version>
-    <rxjava2.version>2.2.4</rxjava2.version>
     <sdk-version>3.0.0</sdk-version>
-    <slf4j.version>1.7.6</slf4j.version>
+    <guava.version>27.0.1-jre</guava.version>
+    <!--<netty-tcnative.version>2.0.25.Final</netty-tcnative.version>-->
```

Comment:
Why commended out properties?

Reply:
They're versions of commented out dependencies or plugins.
If cosmos team needs them, they can enable them back.

Reply:
In general I would rather delete these commented out things over keeping them just in case. We always have source history if someone needs to bring them back. If you don't feel comfortable deleting them then we would at least file a follow-up item to clean them out once you get whatever confirmation you are looking for from the cosmos team.

Reply:
Removed the dependencies.